examples/llm/train_clm/train_clm.py

```diff
@@ -13,20 +13,29 @@
                                       build_dataloader, build_logger,
                                       build_optimizer, build_scheduler)
 from examples.common.config_utils import log_config, update_batch_size_info
-from examples.llm.src.model_registry import COMPOSER_MODEL_REGISTRY
+from examples.llm.src.huggingface.hf_causal_lm import ComposerHFCausalLM
+from examples.llm.src.mosaic_gpt.model import ComposerMosaicGPT
 
 
 def build_composer_model(cfg):
-    warnings.filterwarnings(
-        action='ignore',
-        message='Torchmetrics v0.9 introduced a new argument class property')
-    try:
-        return COMPOSER_MODEL_REGISTRY[cfg.name](cfg)
-    except:
+    if cfg.name == 'mosaic_gpt':
+        return ComposerMosaicGPT(cfg)
+    elif cfg.name == 'hf_causal_lm':
+        return ComposerHFCausalLM(cfg)
+    else:
         raise ValueError(f'Not sure how to build model with name={cfg.name}')
```

Comment:
I know this isn't part of the PR, but if you feel like tacking on a minor TODO, we should specify the valid options in these `ValueError`s.

Alternatively, feel free to ignore this comment for now and maybe lets make a separate JIRA to do this throughout `examples`.